Gemfile

```diff
@@ -135,7 +135,7 @@ gem 'oj'
 # Rails 7 CSS and JS Bundling
 gem 'cssbundling-rails', '~> 1.4'
 gem 'jsbundling-rails', '~> 1.3'
-gem 'sprockets-rails', require: 'sprockets/railtie'
```

Comment:
I've swapped out Sprockets for Propshaft because we only need fingerprinting and asset resolution.

Reply:
I'm happy with this change, I do wonder if it'd be best to do this separately to make it easier to check and rationalise